client/src/routes/Home/styles.ts

```diff
@@ -5,4 +5,7 @@ export default makeStyles(() => ({
         backgroundColor: "#f6f5f5",
         padding: "2em 0 4em",
     },
+    outer: {
+        //TODO: Add this as something in @Home.tsx depends on it
```

Comment:
Sorry I forgot I deleted the styles for outer, it's no longer needed so I suggest you delete it from here and also delete its usage in the Home.tsx